devnets/3030/lighthouse/deploy_block.txt

```diff
@@ -0,0 +1 @@
+0
```

Comment:
For mainnet and testnet `deploy_block.txt` is missing, it's `0` always.

Reply:
It was already there before this PR, so it is fine